### DISCUSSION TITLE
zh/guide/twikoo

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#450] [Announcements]
# zh/guide/twikoo

一个简洁、安全、免费的静态网站评论系统，基于 腾讯云开发 (https://curl.qcloud.com/KnnJtUom)。 快速上手 部署共有四种方式。 部署方式 描述 --------------------------- -------------------------------------------------------------...

https://plugin-comment2.vuejs.press/zh/guide/twikoo.html

**Comment by ghost** [comment]
感谢！正在使用Twikoo。
